### DISCUSSION TITLE
Crackling at start and stop of connecttospeech

### DISCUSSION THREAD

**Discussion by PrinsP75** [schreibfaul1/ESP32-audioI2S#317] [Q&A]
First a big thanks for making this library because it is making possible to make my audio project with the ESP32. 

All is working fine but I still have one issue with crackling noise at the start and stop of connecttospeech. It's just a split of a second but defenitely audible. For audiostreams it is not really an issue because normally you listen to a stream for a longer time and then you hardly notice it, but I'm also using short texts to navigate through an audible menu (no screen in this project) and for these short texts it is getting a bit annoying to keep hearing those cracks and pops. In between all audio is fine and clear, it is just the initialisation and ending phase of that short audio stream.

It is not a deal breaker for me, but it would be nice if it is possible to get rid of those audible markers at the beginning and end of the spoken texts. If anyone has an idea how to fix this I would love to hear.

Thanks for your time.

**Comment by schreibfaul1** [comment]
Hello PrinsP75,
the crackling could be caused by the PSRAM. There is a design flaw on the chip that cannot be completely eliminated even with the provided fix. Do you have the cracking even without PSRAM?

**Reply by PrinsP75** [reply]
Hi, yes PSRAM is disabled in the settings of the Arduino IDE.
